### DISCUSSION TITLE
Latest Tenk Contract Type Errors

### DISCUSSION THREAD

**Discussion by tomvanriet** [TENK-DAO/tenk#92] [General]
Hey Willem,

So I went through the discussions and found a similar error, you suggested using `yarn deploy:testnet --accountId xx.testnet`

However, after doing this, I still get:
`TypeError: account.createTransaction is not a function
    at Object.main (/mnt/c/Git/tenk/scripts/deploy.ts:66:6)
    at async Object.handler (/mnt/c/Git/tenk/node_modules/near-cli/commands/repl.js:41:17)`

Any suggestions?
Thanks

**Comment by willemneal** [comment]
What is your package.json look like?  The error is because I'm using a newer version of near-api-js (been waiting for them to merge the transaction API `account.createTransaction` (very handy and easy to merge)), so it fails when near-cli doesn't pick up this version.

I would try deleting your node_modules and doing `yarn install --force`.  If this doesn't help let me know.

**Reply by tomvanriet** [reply]
Thanks Willem, I ended up re-cloning the repo and it seemed to work. Your solution probably would do too.
